src/lib/graphqlApiClient.js

```diff
@@ -1,9 +1,14 @@
-import { createClient } from "urql";
+/**
+ * using require syntax  here because graphqlClient is alos used
+ * within the nodejs scripts
+ */
+require("isomorphic-unfetch");
```

Comment:
i heard next.js now ships with server-side fetch

Reply:
since scripts is not part of the nextjs app build I find it may be safer to have an explicit dependency (I will test if it work without)